notes.md

```diff
@@ -51,6 +51,10 @@
 * The total number of shares outstanding is the difference between a cap and `token.totalSupply()` and is rightly a function of the `CrowdsaleContract`.
 * The date the share was purchased is a property of the block the purchase was processed in so there is no need to store it in the token. However the user database might wish to record that information as well for convenience.
 * The price the user paid is a function of the ETH price per Token (which can change during the crowdsale) and ETH price at that point (which can also be set in the `CrowdsaleContract`) and the number of ETH spent, which the `CrowdsaleContract` also knows and which is returned in the transaction receipt.
+* Tokens can only be minted in whole numbers so `floor` the conversion and put the remainder, less some gas, into a `RefundVault`.
+* Move the ETH to be turned into Tokens into the Crowdsale's `wallet`.
```

Comment:
Can we configure the crowdsale wallet to automatically forward to a multisig wallet controlled by cofounders to protect it? Like this: https://blog.zeppelin.solutions/handling-crowdsale-funds-securely-c348467da74c

Reply:
Looks like in the recent implementation of the crowdsale token they just forward it to the beneficiary.  Not sure I understood why that article recommended a timelock; seems unnecessary

Reply:
I've changed this now to base out crowdsale on the following base contracts

* `CappedCrowdsale`,
* `RefundableCrowdsale`, and
* `MintedCrowdsale`

I need to make a number of overrides to that base however to suit our needs.